www-apache/mod_security/mod_security-2.9.6.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
2022

Reply:
Agreed. Just working out the correct process to resolve this conversation

Reply:
Updated